tss-esapi/tests/integration_tests/attributes_tests/nv_index_attributes_tests.rs

```diff
@@ -0,0 +1,324 @@
+// Copyright 2020 Contributors to the Parsec project.
```

Comment:
```suggestion
// Copyright 2021 Contributors to the Parsec project.
```

Reply:
This is a rename from ```tss-esapi/tests/nv_storage_nv_index_attributes_tests.rs``` to ```tss-esapi/tests/integration_tests/attributes_tests/nv_index_attributes_tests.rs``` with few minor modifications so the 2020 is correct here.